pkg/scheduler/scheduler.go

```diff
@@ -569,7 +569,7 @@ func (sched *Scheduler) scheduleOne(ctx context.Context) {
 			// No nodes available is counted as unschedulable rather than an error.
 			metrics.PodScheduleFailures.Inc()
 		} else {
-			klog.Errorf("error selecting node for pod: %v", err)
+			klog.ErrorS(err, "error selecting node for Pod")
```

Comment:
why not add the pod object like the other log message?

Reply:
It seems `err` is a special key in the new log format.

BTW: I'm a little surprised this line is on the top-2 list.

Reply:
This log line was 20th line that generated most logs. For testing we used `gce-master-scale-performance` test. They run 100 nodes load tests. It could be that during bootstrap or scheduling cluster loader pods there are a lot of errors.

Reply:
can we get access to those log message to see what the exact err was?

Reply:
shouldn't we be passing the object (pod in this case) as well?

Reply:
From log file on my workstation (I assume that job logs were deleted as it was couple of months ago):
```running "VolumeBinding" filter plugin for pod "medium-statefulset-0-30": pod has unbound immediate PersistentVolumeClaims```

Reply:
I think that adding `pod` key to this log would be a nice improvement.

Reply:
Thanks! 

@cofyc we should change this to report an unschedulable error rather than an internal error.

Reply:
you're right, it should be an unschedulable error (and unresolvable as preemption would not change anything).

opened an issue for this https://github.com/kubernetes/kubernetes/issues/91755